plugins/org.eclipse.elk.core/src/org/eclipse/elk/core/util/nodespacing/internal/cellsystem/ContainerArea.java

```diff
@@ -0,0 +1,26 @@
+/*******************************************************************************
+ * Copyright (c) 2017 Kiel University and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *    Kiel University - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.elk.core.util.nodespacing.internal.cellsystem;
+
+/**
+ * Enumeration of the three possible cells inside container cells.
```

Comment:
Nope, a `StripContainerCell` hosts one row (or column) of three cells. A `GridContainerCell` hosts a matrix of 3x3 cells. How many children can be hosted is not fixed by the `ContainerCell` super class, though. I'll change the wording of the comment accordingly.